markup.go

```diff
@@ -170,7 +170,19 @@ func If(cond bool, markup ...MarkupOrComponentOrHTML) MarkupOrComponentOrHTML {
 	if cond {
 		return List(markup)
 	}
-	return nil
+	var result []MarkupOrComponentOrHTML
+	for _, m := range markup {
+		if m == nil {
+			continue
+		}
+		switch m.(type) {
+		case *HTML, Component:
+			result = append(result, Tag("noscript"))
+		default:
+			return nil
```

Comment:
I don't think `true` is a valid type for `MarkupOrComponentOrHTML` anyway, is it?

The idea was that only `Component` and `HTML` should return tags, but `Markup` should just return `nil` immediately.  If users are mixing the latter with the former, they'll find out they screwed up when their condition matches I guess, sames as if they just put the wrong mix outside `If`.